src/lib/components/workflow/workflows-summary-configurable-table.svelte

```diff
@@ -19,7 +19,7 @@
 
   $: ({ namespace } = $page.params);
   $: columns = $workflowTableColumns?.[namespace] ?? [];
```

Comment:
Should we add this here too?
```suggestion
  $: columns = $workflowTableColumns?.[namespace] ?? getDefaultColumns();
```